### PR TITLE
Change repo for Beelan Library

### PR DESCRIPTION
**Pull request by sabas1080**
ElectronicCat now supports the Beelan library

the link has changed

Thnaks

### CODE REVIEW

**Comment by per1234** [comment]
Hello @sabas1080. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by sabas1080** [comment]
@per1234 Thanks

**Comment by umbynos** [comment]
Solved in https://github.com/arduino/library-registry/pull/1633#issuecomment-1204110813

**Review by umbynos — APPROVED** [review]

**Pull request merged**
